src/components/board/Stack.vue

```diff
@@ -51,6 +54,19 @@
 			</Actions>
 		</div>
 
+		<Modal v-if="modalShow" title="Archive all cards in this list" @close="modalShow=false">
+			<div class="modal__content">
+				<h3>Archive all cards in this list</h3>
+				<progress :value="archiveAllCardsProgress" :max="cardsByStack.length" />
```

Comment:
cardsByStack will decrease during the process, so ideally you store the max in a local data field before starting the process